frontend/elements/src/entry/popup/popup-logout.ts

```diff
@@ -0,0 +1,52 @@
+import { MEDIA_UI } from '@utils/path';
```

Comment:
entry/popup isn't the right location - should either be a sub-component of some other entry (like home), a common widget, or in core